src/components/sections/HeroSection.tsx

```diff
@@ -0,0 +1,39 @@
+import Heading from "../../../public/hero/HeroHeading.svg"
+import Clouds from "../../../public/hero/HeroClouds.png"
+import Pillars from "../../../public/hero/HeroPillarsCropped.png"
```

Comment:
the same with images, when using public ones, just use them directly in the component